src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -1075,27 +1082,21 @@ void goto_instrument_parse_optionst::instrument_goto_program()
     do_indirect_call_and_rtti_removal();
 
     status() << "Partial inlining" << eom;
-    goto_partial_inline(goto_functions, ns, ui_message_handler, true);
+    goto_partial_inline(goto_functions, ns, ui_message_handler, 0, true);
 
     goto_functions.update();
     goto_functions.compute_loop_numbers();
   }
 
-  // now do full inlining, if requested
-  if(cmdline.isset("inline"))
```

Comment:
Am I reading this right? There were two identical blocks for dealing with `cmdline.isset("inline")`?